scala-package/core/src/test/scala/ml/dmlc/mxnet/TestUtil.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ml.dmlc.mxnet
+
+class TestUtil {
+
+  /**
+    * Allow override of data path. Default is <current directory>/data
+    * @return Data direcotry path ()may be relative)
+    */
+  def getDataDirectory: String = {
+    var dataDir = System.getenv("MXNET_DATA_DIR")
+    if(dataDir == null) {
```

Comment:
add space after `if`, otherwise scala lint will fail.
and I prefer this:
```scala
val dataDir = ...
if (dataDir == null || dataDir.isEmpty) "data"
else dataDir
```